File: 4ejercicio.js
/*
Enunciado: Haz un conteo desde 10 hasta 0 y al final imprime "¡Despegue!".
Requerimientos: Usa un bucle while, el operador -- y un if final.
*/
let contador = 10;

while (contador >= 0) {
	console.log(contador);
	contador--;
}

if (contador < 0) {
	console.log("¡Despegue!");
}

File: 4ejercicio.py
"""Enunciado: Cuenta cuántas vocales tiene una frase ingresada por el usuario.
Requerimientos: Recorre el string con un for, usa un if con el operador in y mantén un contador."""

frase = input("Ingrese una frase: ")
contador_vocales = 0
vocales = "aeiouAEIOU"
for letra in frase:
    if letra in vocales:
        contador_vocales += 1
print("La cantidad de vocales en la frase es:", contador_vocales)
